models/Passenger.js

```diff
@@ -10,11 +10,14 @@ var Passenger = new keystone.List('Passenger', {
 });
 
 Passenger.add({
-    name: { type: String, initial: true, required: true },
-    phone: { type: String, initial: true, required: true },
-    gcm_id: { type: String },
-    direction: { type: Types.Select, options: 'to, from, both', required: true, initial: true},
-    gender: { type: Types.Select, numeric: true, emptyOption: true, options: [{ value: 0, label: 'Unknown' }, { value: 1, label: 'Male' }, { value: 2, label: 'Female' }, { value: 9, label: 'Not Applicable' }] }
+    name: { label: 'Name', type: String, initial: true, required: true },
+    phone: { label: 'Phone', type: String, initial: true, required: true },
+    gcm_id: { type: String, hidden: true },
+    location: { type: Types.Location, defaults: { country: 'USA' } },
+    direction: { label: 'Direction', type: Types.Select, options: 'to, from, both', required: true, initial: true},
+    gender_pref: { label: 'Gender Preference', type: Types.Select, numeric: true, emptyOption: true, options: [{ value: 0, label: 'Unknown' }, { value: 1, label: 'Male' }, { value: 2, label: 'Female' }, { value: 9, label: 'Not Applicable' }] },

```

Comment:
@esolum, @tydahlwave, and @landongerrits Can one of you check if changing the name of this field is going to cause issues with the iOS app? I just put the change into the dev branch but I wanted to check on that before I thought about pushing it to master.